docs/marketplace/fastlane/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Fastlane Marketplace
+description: Appcircle fastlane plugin allows you to distribute your builds to testers directly pipeline.
+tags:
+  [
+    github-marketplace,
+    testing-distribution,
+    ipa distribution,
+    apk distribution,
+    binary distribution,
+  ]
+sidebar_position: 2
+---
+
+Enhance power of distributing the builds to test groups or release beta version of your apps with using **Appcircle Testing Distribution** and **Appcircle Enterprise Store** plugins
+
+### Testing Distribution
+
+It is a process distributing test builds to designated test groups or individuals.
```

Comment:
It is a process of distributing test builds to designated test groups or individuals.